console/v1/types_console_plugin.go

```diff
@@ -48,6 +48,84 @@ type ConsolePluginSpec struct {
 	// i18n is the configuration of plugin's localization resources.
 	// +optional
 	I18n ConsolePluginI18n `json:"i18n"`
+	// contentSecurityPolicy is a list of Content Security Policy directives for the plugin.
+	// Each directive specifies a list of values that indicate server origins and script endpoints
+	// from which the plugin's assets can be loaded.
+	// This helps guard against cross-site scripting (XSS) attacks.
+	// Available directive types are default-src, script-src, img-src, style-src and font-src.
```

Comment:
```suggestion
	// Available directives are default-src, script-src, img-src, style-src and font-src.
```

Reply:
I'd also note that by default console adds the value `self` for the various `src` directives.